src/libs/hashPassword.js

```diff
@@ -0,0 +1,12 @@
+const argon2 = require('argon2');
+
+const hashPassword = async(password) => {
```

Comment:
can you pls share link to articale, where you have found argon? It's seems interesting to me